openedx/features/philu_courseware/helpers.py

```diff
@@ -87,4 +85,25 @@ def get_pre_post_assessments_score(user, course_id, chapter_id):
         'pre_assessment_score': pre_assessment_score,
         'post_assessment_score': post_assessment_score,
         'pre_assessment_attempted': pre_assessment_attempted,
+        'all_pre_assessment_attempted': all_pre_assessment_attempted,
+        'all_post_assessment_attempted': all_post_assessment_attempted,
     }
+
+
+def is_all_attempted(section):
+    attempted_problem_scores = [score for score in section.problem_scores.values()
+                                    if score.first_attempted] \
+                                    if section and section.problem_scores else []
+    return len(attempted_problem_scores) == constants.COMPETENCY_ASSESSMENT_PROBLEMS_COUNT
```

Comment:
`COMPETENCY_ASSESSMENT_PROBLEMS_COUNT` can we make this configurable from django admin? i.e. put it in site_configurations and if the setting is not present in the site_configurations we can the default value which will be 5. Reason being that one of the major benefits of the hybrid approach is that it doesn't limit you to a certain number of questions

Reply:
done

Reply:
@saad-waqar please review it.